### PR TITLE
Create Orders View

### LINKED ISSUE #96: Complete checkout backend

### LINKED ISSUE #103: Create OrderListView component

### PR DESCRIPTION
**Pull request by gynnantonix**
Depends on #102 
Creates an Orders view and, for logged-in users, wires it into the frontend route `/orders`.  Adds a Orders link to the nav.

Closes #103

### CODE REVIEW

File: client/components/OrdersView.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+ * Renders info pane for chosen order
+ */
+const OrderDetails = ({order}) => (
+    <div className="card col">
+        <div className="card-header">
+            Order Details (id #{order.id})
+        </div>
+        <div className="card-body">
+            <h5 className="card-title">{formatOrderDate(order.updatedAt)}</h5>
+            <p className="card-text">...order details go here...</p>
```
Comment (Scutellicious) [inline client/components/OrdersView.js:30]: is this order details go here line supposed to be in here??

**Review by Scutellicious — COMMENTED** [review]
just would love a review of two comments before merge

**Review by Scutellicious — APPROVED** [review]
better late than never!

File: client/components/OrdersView.js (hunk 1 above)
Comment (gynnantonix) [reply client/components/OrdersView.js:30]: I didn't get around to laying out the actual order metadata

**Pull request merged**